ruby/build_release.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+set -ex
+
+WORKING_DIR=$(dirname $(readlink $BASH_SOURCE))
+
+# Move all files to the correct directory
+cp src/google/protobuf/*.rb $WORKING_DIR/lib/google/protobuf
+cp ruby/protobuf_java_release-project.jar $WORKING_DIR/lib/google/protobuf_java.jar
+UTF8_DIR=$WORKING_DIR/ext/google/protobuf_c/third_party/utf8_range
+mkdir -p $UTF8_DIR
+cp external/utf8_range/LICENSE $UTF8_DIR/LICENSE
+cp external/utf8_range/naive.c $UTF8_DIR
+cp external/utf8_range/range2-neon.c $UTF8_DIR
+cp external/utf8_range/range2-sse.c $UTF8_DIR
+cp external/utf8_range/utf8_range.h $UTF8_DIR
+
+# Make all files global readable/writable/executable
+cd $WORKING_DIR
+chmod -R 777 ./
+
+# Build gem
+bash --login -c "rvm use system && gem build google-protobuf.gemspec"
```

Comment:
"system" could be jruby couldn't it?

Reply:
I guess it could, what should I set this to to get a reasonable ruby version?

Reply:
I would leave it alone and just require the caller to set the ruby version.  The future plan for Bazel+Ruby is that our ruby rules will use the system ruby by default, with an optional flag to pin a specific version.  So we could just run this twice, once with cruby and once with jruby

Reply:
Leave it alone meaning don't call `rvm use` and just call `gem build google-protobuf.gemspec`? Isn't that also just defaulting to the system version?

Reply:
Exactly.  It defaults to the *current* version (which is distinct from rvm's "system" I believe), but it doesn't make any assumptions about what that version is.  So if we want to produce both c-ruby and java-ruby artifacts, we'd use rvm to set our version and run this target twice

Reply:
Resolved offline. Will wait to submit until we have implemented a flag to specify ruby versions.